.github/workflows/ci.yml

```diff
@@ -22,14 +22,7 @@ jobs:
         command: ['lint', 'lint:attw', 'lint:monorepo', 'lint:pub', 'test', 'test:tsd', 'type:check', 'build']
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v2
-        with:
-          version: 8.12.0
-      - uses: actions/setup-node@v4
-        with:
-          cache: 'pnpm'
-          cache-dependency-path: 'pnpm-lock.yaml'
-          node-version-file: '.nvmrc'
+      - uses: ./.github/pnpm/setup-node
```

Comment:
@minsoo-web why is your suggestion better than this? I need more reason to change it